modules/claims_api/app/controllers/claims_api/concerns/itf_verification.rb

```diff
@@ -5,21 +5,6 @@ module ItfVerification
     extend ActiveSupport::Concern
 
     included do
-      def verify_itf
-        active = itf_service.get_active('compensation')
-        if !active['intent_to_file'] || active['intent_to_file'].expiration_date < Time.now.utc
-          error = {
-            errors: [
-              {
-                status: 422,
-                details: 'Intent to File Expiration Date not valid, resubmit ITF.'
-              }
-            ]
-          }
-          render json: error, status: :unprocessable_entity
-        end
-      end
-
```

Comment:
So since we're removing the verification portion of this, maybe it should be refactored or at least renamed?

Reply:
I removed it